exercises/exercise_session_1.tex

```diff
@@ -0,0 +1,57 @@
+% Include the header of the document.
+\input{template_header}
+
+\title{Large-Scale Distributed Systems: Exercise Session 1}
+\maketitle
+
+\section{Introduction}
+\label{sec:introduction}
+
+The core of any distributed system is a set of distributed algorithms that ensures the behaviour and the correctness of the global state. In contrast to single monolithic systems, distributed systems have usually the advantage that other processes are conducting their computations \emph{independently} (asynchronously) from other \emph{entities}, i.e., a different machine or node. However, in order to ensure the correctness of the global goal, information exchange is required. The difficulty in this setting arises from the fact that a physical communication medium is intrinsically lossy due to the fact that other (physical) processes influence the communication signal. Nevertheless, if we construct our infrastructure in such a way that we can minimize the loss, we can assume that with a certain probability some form of \emph{useful} communication might occur between two or more entities, i.e., sometimes, the medium allows for information to pass without being destroyed. In the lecture we saw that using this principle, we can construct relatively complex abstractions, and more importantly, build on top of previous abstractions (layering) to ensure that information always arrives once it is send.\\
+
```

Comment:
You should maybe also comment that processes may crash, in addition to the communication systems. Still, we want a system that remains reliable even if some of nodes crash.